gpkit/constraints/model.py

```diff
@@ -90,6 +75,15 @@ def __init__(self, cost=None, constraints=None, *args, **kwargs):
             # even if they aren't used in any constraints
             self.unused_variables = setup_vars
             self.reset_varkeys()
+        # for backwards compatibility keep add_modelnames
+        if not hasattr(self, "setup") and self.__class__.__name__ != "Model":
```

Comment:
would be nice to have an inline comment here explaining the plan for when this will be removed